tests/lib/journalctl.sh

```diff
@@ -11,7 +11,7 @@ get_last_journalctl_cursor(){
 }
 
 start_new_journalctl_log(){
-    echo "New test starts here - $SPREAD_JOB" | systemd-cat -t snapd-test
+    echo "New test starts here - $SPREAD_JOB" | systemd-cat -t snapd-test -p alert
```

Comment:
AFAICT we don't need the changes in this file - this is all related to systemd/systemd#11502 it seems